### PR TITLE
Uses name instead of menu.

### PR DESCRIPTION
**Pull request by evanrs**

### CODE REVIEW

File: src/AppBar/Navigation.jsx [hunk 1]
```diff
@@ -60,6 +60,8 @@ class AppNav extends React.Component {
 
     render() {
         const {user, config} = this.props;
+        console.log(user);
```
Comment (tholex) [inline src/AppBar/Navigation.jsx]: log

**Comment by diezguerra** [comment]
@evanrs is there a reason to hold this?

**Comment by tholex** [comment]
This was held until we have a tooltip component or some way of showing that there's a menu there (on hover is fine)

**Comment by evanrs** [comment]
Closing stale branch.

**Pull request closed without merging**